### PR TITLE
[rabbitmq] Add disk_free_limit param to config

### PR DESCRIPTION
**Pull request by adanin**
PRD-1878
Tested by hands.

### CODE REVIEW

File: deployment/puppet/rabbitmq/templates/rabbitmq.config [hunk 1]
```diff
@@ -7,7 +7,8 @@
     {inet_default_connect_options, [{nodelay,true}]}
   ]}
   ,{rabbit, [
-    {tcp_listeners, [<%= port %>]}
+    {tcp_listeners, [<%= port %>]},
+    {disk_free_limit, <%= disk_free_limit %>}
```
Comment (bogdando) [inline deployment/puppet/rabbitmq/templates/rabbitmq.config:5]: It should be {disk_free_limit, [<%= disk_free_limit %>]} , AFAIK

Comment (adanin) [inline deployment/puppet/rabbitmq/templates/rabbitmq.config:5]: No. It's a single value parameter. TCP_listeners is a multi value parameter so it is a list.

**Comment by mihgen** [comment]
rebase please. How did you test this?

**Comment by adanin** [comment]
Fixed and rebased. 
Tested by hands. I copied manifests to the master node and made 'puppet apply' on it. Rabbitmq.config became

```
[root@fuel ~]# cat /etc/rabbitmq/rabbitmq.config 
% This file managed by Puppet 2.7.19
% Template Path: rabbitmq/templates/rabbitmq.config
[
  {rabbit, [
    {disk_free_limit, 100000000}
  ]}
  ,{rabbitmq_stomp, [
    {tcp_listeners, [61613]}
  ]}
].
% EOF
```

Rabbitmq-server restarted successfully. Rabbitmqctl output was:

```
[root@fuel ~]# rabbitmqctl status
Status of node rabbit@fuel ...
[{pid,5693},
....................
 {disk_free_limit,100000000},
 {disk_free,17105162240},
....................
```

Please review.

**Comment by mihgen** [comment]
Please get it reviewed by Fuel Library team, and then we can merge.

**Pull request closed without merging**